src/main/java/com/org/jenkins/custom/jenkins/distribution/service/util/Util.java

```diff
@@ -39,7 +44,24 @@ public void cleanupTempDirectory(File file) {
         }
     }
 
-    public JSONObject convertPayloadToJSON(String payload) throws Exception{
+    public JSONObject convertPayloadToJSON(String payload) {
```

Comment:
I think this would still throw an exception?

Reply:
Oh Intellij told me that was redundant so I removed it :)